### ISSUE TITLE
Error using l-popup component.

### ISSUE THREAD

**Issue by pablocattaneo** [schlunsen/nuxt-leaflet#20]
Hi there! First of all thanks for this great module.

I have this  problems using l-popup component when click over a point:

![image](https://user-images.githubusercontent.com/11772956/55179301-ea8a7500-5165-11e9-96c7-e8e455696f15.png)

Here is the code I have written in a page.

```
<template>
  <no-ssr>
    <l-map style="height: 100vh; width: 100%" :zoom="zoom" :center="center">
      <l-tile-layer :url="url"></l-tile-layer>
      <l-marker :lat-lng="markerLatLng">
        <l-popup><span>Hello!</span></l-popup>
      </l-marker>
    </l-map>
  </no-ssr>
</template>

<script>
export default {
  head() {
    return {
      link: [
        {
          rel: "stylesheet",
          href:
            "https://cdnjs.cloudflare.com/ajax/libs/leaflet.markercluster/1.4.1/MarkerCluster.Default.css"
        },
        {
          rel: "stylesheet",
          href:
            "https://cdnjs.cloudflare.com/ajax/libs/leaflet.markercluster/1.4.1/MarkerCluster.css"
        }
      ]
    };
  },
  data() {
    return {
      url: "http://{s}.tile.osm.org/{z}/{x}/{y}.png",
      zoom: 8,
      center: [47.41322, -1.319482],
      markerLatLng: [47.31322, -1.319482]
    };
  },
  mounted() {
    // eslint-disable-next-line no-console
    console.log(this.$L);
  }
};
</script>

```

package verions:

    "nuxt": "^2.5.1",
    "nuxt-leaflet": "0.0.15",

**Comment by schlunsen** [comment]
See updated version

**Comment by pablocattaneo** [comment]
Great! after update it works perfectly. Thank you!

I would like to propose a improvement, before write this issue I ran `npm update` to get an update version, but nothing happened then run `npm outdate` and notice that there was a new version but in package.json I had this line:

`"nuxt-leaflet": "0.0.15",`

So I was never going to get a new version of the module, at least I changed it, so I changed it to `^0.0.x` to get the new version, run `npm update` and now a I have the last version and the problem was result but in my package.json now a have:

`"nuxt-leaflet": "0.0.18"`

What it mean that when a new version release I won't be alert when run "npm outdate". It showl be great if we can update this package using `npm update`
